setup/2. git.md

```diff
@@ -0,0 +1,37 @@
+# Git and Github
+
+## Git
+
+Git is a version control software, which allows you to save your code in steps, like different drafts for an essay. It also allows for branches, which could represent someone else's edits to your paper, which then you merge into your main paper.
+
+For this tutorial, git and Github are not necessary, but highly recommended. You can download git from: [Git-SCM](https://git-scm.com/).
+
+For a tutorial on how git works, this [Git Tutorial](https://www.tutorialspoint.com/git/) is a useful resource.
+
+## Github
+
+Github is an online service that allows people to store their code online for free. Github uses the git program for this. You can create an account, and once that is complete, navigate to [The Fabric Example Mod](https://github.com/fabricMC/fabric-example-mod). This example mod created by fabric is template for your mods, with everything setup.
+
+The reason for hosting your code publicly on Github has many reasons. Fabric believes in Free and Open Source Software (FOSS), meaning that anyone can see and edit the code. This does not allow people to steal your code, but allows them to use it to improve their own code. You can select a license which tells others how your code can be used. See [this site](https://choosealicense.com/) as a way to select a good license. The MIT license is a common license for modders. Hosting your code on Github also allows other users to add content to your mods that you might not be able to do, such as translations and detect and report issues with your mod, like a crash. 
```

Comment:
> `See [this site](https://choosealicense.com/) [...]`

You shouldn't just call it "this site", as people won't know what site it is. Maybe `See [choosealicense.com](https://choosealicense.com/)`?